docs/source/apidoc.md

```diff
@@ -7,331 +7,334 @@
+#### Noise Scaling: Unitary Folding
 ```{eval-rst}
-.. automodule:: mitiq.ddd.rules.rules
+.. automodule:: mitiq.zne.scaling.folding
    :members:
 ```
 
-## Executors
+#### Noise Scaling: Identity Insertion Scaling
 ```{eval-rst}
-.. automodule:: mitiq.executor.executor
+.. automodule:: mitiq.zne.scaling.identity_insertion
    :members:
 ```
 
-## Observables
-
-### Observable
+#### Noise Scaling: Layerwise Folding
 ```{eval-rst}
-.. automodule:: mitiq.observable.observable
+.. automodule:: mitiq.zne.scaling.layer_scaling
    :members:
 ```
 
-### Pauli
+#### Noise Scaling: Parameter Calibration
 ```{eval-rst}
-.. automodule:: mitiq.observable.pauli
+.. automodule:: mitiq.zne.scaling.parameter
    :members:
 ```
 
-## Probabilistic Error Cancellation
 
-### Probabilistic Error Cancellation (High-Level Tools)
+## Tools For Error Mitigation Techniques
+
+### Benchmarks
+
+#### GHZ Circuits
 ```{eval-rst}
-.. automodule:: mitiq.pec.pec
+.. automodule:: mitiq.benchmarks.ghz_circuits
    :members:
 ```
 
-### Quasi-Probability Representations
+#### Mirror Circuits
+
 ```{eval-rst}
-.. automodule:: mitiq.pec.representations.optimal
+.. automodule:: mitiq.benchmarks.mirror_circuits
    :members:
+```
 
-.. automodule:: mitiq.pec.representations.damping
-   :members:
 
-.. automodule:: mitiq.pec.representations.depolarizing
+#### Mirror Quantum Volume Circuits
+```{eval-rst}
+.. automodule:: mitiq.benchmarks.mirror_qv_circuits
    :members:
 ```
 
-### Learning-based PEC
+#### Quantum Phase Estimation Circuits
 ```{eval-rst}
-.. automodule:: mitiq.pec.representations.biased_noise
+.. automodule:: mitiq.benchmarks.qpe_circuits
    :members:
+```
 
-.. automodule:: mitiq.pec.representations.learning
+
+#### Quantum Volume Circuits
+```{eval-rst}
+.. automodule:: mitiq.benchmarks.quantum_volume_circuits
    :members:
 ```
 
-### Sampling from a Noisy Decomposition of an Ideal Operation
+
+#### Randomized Benchmarking Circuits
+
 ```{eval-rst}
-.. automodule:: mitiq.pec.sampling
+.. automodule:: mitiq.benchmarks.randomized_benchmarking
    :members:
 ```
 
-### Probabilistic Error Cancellation Types
+#### Rotated Randomized Benchmarking Circuits
+
 ```{eval-rst}
-.. automodule:: mitiq.pec.types.types
+.. automodule:: mitiq.benchmarks.rotated_randomized_benchmarking
    :members:
 ```
 
-### Utilities for Quantum Channels
+
+#### W State Circuits
 ```{eval-rst}
-.. automodule:: mitiq.pec.channels
+.. automodule:: mitiq.benchmarks.w_state_circuits
    :members:
 ```
 
-## Raw
+### Calibration
 
-### Run experiments without error mitigation (raw results)
 ```{eval-rst}
-.. automodule:: mitiq.raw.raw
+.. automodule:: mitiq.calibration.calibrator
+   :members:
+
+.. automodule:: mitiq.calibration.settings
+   :members:
 ```
 
-## Readout-Error Mitigation
 
-### Postselection
+### Observables
+
+#### Observable
 ```{eval-rst}
-.. automodule:: mitiq.rem.post_select
+.. automodule:: mitiq.observable.observable
    :members:
 ```
 
-### REM Technique
+#### Pauli
```

Comment:
Pauli Strings? Pauli Operators? I feel like it needs something after Pauli, otherwise it sounds weird. Might just be me though!